modules/http-helper/http_helper.go

```diff
@@ -13,19 +14,23 @@ import (
 	"github.com/gruntwork-io/terratest/modules/retry"
 )
 
-// HttpGet performs an HTTP GET on the given URL and return the HTTP status code and body. If there's any error, fail the test.
-func HttpGet(t *testing.T, url string) (int, string) {
-	statusCode, body, err := HttpGetE(t, url)
+// HttpGet performs an HTTP GET, with an optional pointer to a custom TLS configuration, on the given URL and
+// return the HTTP status code and body. If there's any error, fail the test.
+func HttpGet(t *testing.T, url string, TLSConfig *tls.Config) (int, string) {
+	statusCode, body, err := HttpGetE(t, url, TLSConfig)
 	if err != nil {
 		t.Fatal(err)
 	}
 	return statusCode, body
 }
 
-// HttpGetE performs an HTTP GET on the given URL and return the HTTP status code, body, and any error.
-func HttpGetE(t *testing.T, url string) (int, string, error) {
+// HttpGetE performs an HTTP GET, with an optional pointer to a custom TLS configuration, on the given URL and
+// return the HTTP status code, body, and any error.
+func HttpGetE(t *testing.T, url string, TLSConfig *tls.Config) (int, string, error) {
```

Comment:
It is unconventional in go to name local variables starting with caps. Use `tlsConfig`.